Trading/Mode/dip_analyser_trading_mode.py

```diff
@@ -0,0 +1,397 @@
+"""
+OctoBot Tentacle
+
+$tentacle_description: {
+    "package_name": "OctoBot-Tentacles",
+    "name": "dip_analyser_trading_mode",
+    "type": "Trading",
+    "subtype": "Mode",
+    "version": "1.1.0",
+    "requirements": ["dip_analyser_strategy_evaluator"],
+    "config_files": ["DipAnalyserTradingMode.json"],
+    "tests":["test_dip_analyser_trading_mode"]
+}
+"""
+
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+from ccxt import InsufficientFunds
+
+from config import START_PENDING_EVAL_NOTE, TraderOrderType, TradeOrderSide
+from trading.trader.modes.abstract_mode_creator import AbstractTradingModeCreator
+from trading.trader.modes.abstract_mode_decider import AbstractTradingModeDecider
+from trading.trader.modes.abstract_trading_mode import AbstractTradingMode
+from evaluator.Strategies import DipAnalyserStrategyEvaluator
+from tools.symbol_util import split_symbol
+
+
+class DipAnalyserTradingMode(AbstractTradingMode):
+    DESCRIPTION = "DipAnalyserTradingMode is a trading mode adapted to volatile markets. It will look for local " \
+                  "market bottoms, weight them and buy these bottoms. It never sells except after a buy order is " \
+                  "filled. When a buy order is filled, sell orders will automatically be created at a higher price " \
+                  "than this of the filled buy order. The number of sell orders created after each buy is specified " \
+                  "in DipAnalyserTradingMode.json. A higher risk will make larger buy orders. Make sure to have the " \
+                  "base of each traded pair to be traded otherwise the DipAnalyserTradingMode won't trade: it will " \
+                  "never be able to place the initial buy orders."
```

Comment:
What if automatically created sells orders are not filled some months after ?

Reply:
They will remaing in the exchange

Reply:
So if we imagine multiple currencies in the same situation the bot wont be able to trade until reboot or maybe a long time ?

Reply:
Yep, if you make it trade on a completely down market for all of its pairs for a long time, at the end it will be stuck

Reply:
To prevent this, when making a buy order, the trading mode checks how much of this asset is already in its bag and lower the amount accordingly

Reply:
Ok so I think we can just add a warning about this specific behaviour in the description

Reply:
Good idea !